src/llm.py

```diff
@@ -101,6 +102,15 @@ def get_random_context() -> List[str]:
 # Get context about the server
 async def get_server_context(message: Message) -> List[str]:
     result = []
+
+    # Detect if song is currently playing
+    if hasattr(message, "guild"):
+        bc = bust.controllers.get(message.guild.id)
+        if bc and bc.is_active():
+            result.append("The bust is going on right now!")
+            result.append(f"Now playing: {bc.current_song()}")
+            result.append("Tell everyone you can't respond since you're busy busting.")
```

Comment:
I'm surprised this doesn't cause her automatic responses to be like "I'm busy" all the time.

Reply:
Yeah that's kind of what I was going for but I think what happened instead was way better